frontend/src/scss/base/_prime-styles.scss

```diff
@@ -1133,13 +1133,6 @@ $results-dropdown-spacing: #{units(4)} - #{units(2)} - 22px - #{units(4)}; // he
   border: 2px solid #dfe1e2;
 }
 
-.usa-checkbox__input:disabled + .usa-checkbox__label::before,
```

Comment:
@mpbrown Removing this should have fixed it! Thank you for catching that!
screenshot of dev6
<img width="411" alt="Screenshot 2024-05-07 at 11 25 20" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/7e848487-0f36-418e-b4dc-f7eda8204da3">